docs/pages/functionality/filter-management.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Filter Management
+permalink: functionality/filter-management
+type: functionality
+language: en
+list: include
+---
+
+### What is an Envoy ?
+
+An <strong>Envoy</strong> is a high performance, programmable L3/L4 and L7 proxy that many mesh services implement like istio. There are network filters at the core of the Envoy’s connection and the traffic handling which when mixed with the filter chains allows the implementation of the higher-order functionalities for access control, transformation, data enrichment, auditing and many more. 
+
+### What is a WASM ?
+
+<strong>WASM (<i>Web Assembly</i>)</strong> is an extremely interesting way which greatly simplifies the process of extending Envoy with new capabilities. WASM is an efficient portable binary instruction format providing an embeddable and isolated execution environment.
+
+### Why are WASM filters used ?
+
+The Wasm filters are used for the following reasons :
+
+-  <strong>Agility</strong> : We can dynamically load into the running envoy without stopping or re-compiling it.
+- <strong>Maintainability</strong> : To extend the functionality we don’t have to change the entire codebase.
+- <strong>Diversity</strong> : WASM filters can be easily compiled with many popular programming languages like C/C++ and Rust.
+- <strong>Reliability and isolation</strong> : WASM filters are deployed into VM and hence they are isolated from the Envoy process. For instance if the WASM filter crashes it will not impact the envoy process.
+- <strong>Security</strong> : The WASM filter uses a well defined API to connect with the host i.e envoy proxy which can be modified only a limited number of connections.
+- <strong>Speed</strong> : It is originally used to speed up large web application
+
+### What are the disadvantages of WASM filters to be taken into consideration ?
+
+The disadvantages of the WASM filters are :
+
+- The performance is nearly 70% as the native C++.
+- It takes a higher memory usage due to the requirement of starting one or more WASM virtual machines.
```

Comment:
The phrasing of this bullet with "It" here is inconsistent phrasing with the other bullet point.

Reply:
Does each filter run in its own WASM VM or does each filter share the same WASM VM?

Reply:
@leecalcote I found that multiple filters share the same WASM VM.